clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -1775,6 +1775,36 @@ inline void segmented_sort_pairs(
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|reduce_argmax|dpct
+// DPCT_DEPENDENCY_EMPTY
```

Comment:
reduce_argmin should depend on dpct::arg_index_input_iterator

Reply:
Thanks, fixed.